sw/device/tests/rv_core_ibex_nmi_irq_test.c

```diff
@@ -0,0 +1,263 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <assert.h>
+#include <limits.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/math.h"
```

Comment:
Math isn't explicitly listed as a dependency. Is it automatically included through something else?

Reply:
I'm dropping the line, is was a copy paste remain.